test/test_implementation/multi_particle_dist.jl

```diff
@@ -0,0 +1,37 @@
+
+PARTICLE_DIRECTIONS = (Incoming(), Outgoing(), QEDevents.UnknownDirection())
+
+struct TestMultiParticleDist{DT<:Tuple,PT<:Tuple,RT} <: MultiParticleDistribution
+    dirs::DT
+    parts::PT
+    function TestMultiParticleDist(dirs::DT, parts::PT) where {DT,PT}
+        res_type = Tuple{QEDevents._assemble_tuple_types(parts, dirs, SFourMomentum)...}
+        return new{DT,PT,res_type}(dirs, parts)
+    end
+end
+
+#Base.length(d::TestMultiParticleDist) = length(d.parts)
```

Comment:
Uncomment or delete this.

Reply:
done in https://github.com/QEDjl-project/QEDevents.jl/commit/31c9fa1212696a29f21e4324cb269ac3c0d70fe0